### ISSUE TITLE
repl ignores inputrc

### ISSUE THREAD

**Issue by nathanielvirgo** [purescript/spago#720]
I have the following lines in my `~/.inputrc`:

```
"\e[A": history-search-backward
"\e[B": history-search-forward
```

For most apps that use the readline library, this means that I can search my history by typing the first few characters of a previous command and pressing the up arrow. However, `spago repl` seems to ignore this, and just goes to the previous history entry, regardless of what I type.

This feature would be particularly useful for purescript, since I think there's no way to reload a package without also losing everything that was entered previously. It would be a lot easier to re-enter commands if I was able to search the history for them.

**Comment by f-f** [comment]
@nathanielvirgo `spago repl` is just a thin wrapper over `purs repl` - are you able to replicate with just `purs repl`?

**Comment by nathanielvirgo** [comment]
I was a bit confused about that. `purs repl` just says:

```
% purs repl  
purs repl: No input files; try running `pulp psci` instead.
For help getting started, visit https://github.com/purescript/documentation/blob/master/guides/PSCi.md
Usage: For basic information, try the `--help' option.
```

**Comment by f-f** [comment]
Right, sorry for that. This should work properly: `spago repl $(spago sources)`

**Comment by f-f** [comment]
@nathanielvirgo have you succeeded at testing if this behaviour is due to `spago` or `purs`?

I dug into this, and it looks like since `purs` is using the `haskeline` library to read input, your `.inputrc` would not work, but you should be able to configure the repl with a `~/.haskeline` file, [more details here](https://github.com/judah/haskeline/wiki/UserPreferences)

**Comment by m15a** [comment]
Encountered a similar problem and I found that
- in the repl launched by `purs repl $(spago sources)`, completion just works, and
- by `spago repl`, it does not work.

In both cases I installed psci-support.

- purs version: 0.15.14
- spago version: 0.93.22
